src/structured-data/handler.js

```diff
@@ -194,6 +195,9 @@ export async function opportunityAndSuggestions(auditUrl, auditData, context) {
 
   // Convert suggestions to errors
   for (const issue of auditData.auditResult.issues) {
+    if (isInternalCustomer && issue.rootType === 'ImageObject') {
```

Comment:
This logic should probably happen earlier in the audit. For example in `processStructuredData` so that the problematic issues don't get stored.

Your code excludes all detected issues for `ImageObject`. However, only one specific error should be excluded. Please make the condition more precise.